charts/hono/templates/artemis/artemis-deployment.yaml

```diff
@@ -86,4 +86,5 @@ spec:
         {{- include "hono.container.volumeMounts" $args | indent 8 }}
       volumes:
       {{- include "hono.pod.volumes" $args | indent 6 }}
+      {{- include "hono.pod.affinity" $args | nindent 6 }}
```

Comment:
Having the `affinity` entry at the bottom of the `spec` items list seems somewhat non-intuitive to me. Could you move it up (somewhere above the (init)containers ?